GiEnJul/ClientApp/src/components/admin/suggestedConnections/Types.tsx

```diff
@@ -0,0 +1,33 @@
+import Gender from "../../../common/enums/Gender";
+
+export interface GiverType {
+  email: string;
+  fullName: string;
+  maxReceivers: Number;
+  rowKey: string;
+}
+
+export interface RecipientType {
+  rowKey: string;
+  partitionKey: string;
+  familyId: string;
+
+  dinner: string;
+  dessert: string;
+  note: string;
+
+  contactFullName: string;
+  contactEmail: string;
+  contactPhoneNumber: string;
+
+  institution: string;
+  referenceId: string;
+
+  familyMembers: PersonType[];
+}
+export interface PersonType {
+  rowKey: string;
+  wish: string;
```

Comment:
Can this be null as well?

Reply:
Strings are nullable, but cannot be undefined. This works properly, as the RecipientTable will display "Giver kjøper alderstilpasset gave" if wish is null.